packages/core-components/src/layout/Sidebar/config.ts

```diff
@@ -20,6 +20,32 @@ const drawerWidthClosed = 72;
 const iconPadding = 24;
 const userBadgePadding = 18;
 
+export type SidebarOptions = {
+  drawerWidthClosed?: number;
+  drawerWidthOpen?: number;
+};
+
+export type SubmenuOptions = {
+  drawerWidthClosed?: number;
+  drawerWidthOpen?: number;
+};
+
+export type SidebarConfig = {
```

Comment:
The configs are internal right? Could you mark these as `/** @internal */` and the options as `/** @public */` just so we avoid confusion/mistakes?

Reply:
Got it, will also keep this in mind for future PRs